src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -103,12 +103,23 @@ def start_service(args):
     def validate_port(port, force_start):
         if is_port_in_use(port):
             if force_start:
-                logger.warning(f"Force restart the service on the port {port}.")
+                message = f"Force restart the service on the port {port}."
+                print(message)
+                logger.warning(message)
                 kill_exist_service(port)
             else:
                 logger.warning(f"Service port {port} is used.")
                 raise UserErrorException(f"Service port {port} is used.")
 
+    if is_run_from_built_binary():
+        # For msi installer, use vbs to start pfs in a hidden window. But it doesn't support redirect output in the
+        # hidden window to terminal. So we redirect output to a file. And then print the file content to terminal in
+        # pfs.bat.
+        old_stdout = sys.stdout
+        old_stderr = sys.stderr
+        parent_dir = os.path.dirname(sys.executable)
+        sys.stdout = open(os.path.join(parent_dir, "output.txt"), "w")
```

Comment:
what about also show the target log file name on console when starting the service?

Reply:
will consider added it when change command to pf service. But not sure just added in show-status is enough?